datastore/tasks.js

```diff
@@ -22,12 +21,16 @@ if (!projectId) {
   throw new Error('TEST_PROJECT_ID environment variable required.');
 }
 
+// [START build_service
+var gcloud = require('gcloud');
+
 var datastore = gcloud.datastore({
   projectId: projectId
 });
+// [END build_service]
 
 /*
-// [START build_service]
+Installation and setup instructions.
```

Comment:
There isn't a great way to do this. However as it was these will all get included as code, rather than markdown. Instead I'll copy your markdown directly into the docs manually.